### ISSUE TITLE
Hoopla integration

### ISSUE THREAD

**Issue by seangirard** [spokanelibrary/ent#22]
Within the record, Freegal is displayed.  Can we do this with Hoopla too?

-janet

**Comment by seangirard** [comment]
It doesn't look like Hoopla exposes any APIs.

We might be able to do some kind of Hoopla promo tile on dvd detail pages.

**Comment by jsmithley** [comment]
We might also want to do a promo tile on/for the graphic novels, too? I know they change (more) frequently but hoopla does have a lot of the popular titles we have in print format. Just a thought.

**Comment by seangirard** [comment]
Yeah good idea. Once we're setup for one adding to other types of content shouldn't be a problem.
